### ISSUE TITLE
CAF: need a C8oRoute evolution for MB lazy loading

### ISSUE THREAD

**Issue by nathalie-lammelin** [convertigo/convertigo#238]
In lazy loading, page components (`C8oPage`) are no more imported in the `app.component.ts`.
Pages are now referenced through their **name**.
Thus, we need  an evolution to declare routes in app as following
```
this.router.addRouteListener(
	new C8oRouteListener(["*"])
		.addRoute(
			new C8oRoute((data:any)=>{return true}, tableOptions)
				.setTarget("push", "Page1")
		)
);
```
**`.setTarget(String, String)`** where second parameter is the page's name,
instead of using `.setTarget(String, C8oPage)`

**Comment by nathalie-lammelin** [comment]
Related with [caf #20](https://github.com/convertigo/c8osdk-angular-caf/issues/20)

**Comment by nathalie-lammelin** [comment]
Implemented since `c8ocaf 1.1.8-beta4`

**Comment by gregory-vorbe** [comment]
OK in **Convertigo EMS Studio 7.7.0_beta (build 06367-develop-7.7.0-beta)**
